src/validators/module/index.js

```diff
@@ -1,8 +1,9 @@
 import {noop} from 'lodash'
+import validateLoadersEntry from './validateLoadersEntry'
+
 export default [
-  {key: 'module.loaders', validate: noop},
-  {key: 'module.preLoaders', validate: noop},
-  {key: 'module.postLoaders', validate: noop},
+  {key: 'module.loaders', validate: validateLoadersEntry},
```

Comment:
:+1: Wanted to conform to `validateEntry` function name convention we have in the codebase. But `validateLoaders` is clearer, agreed. Refactored it.